app/src/components/Forms/SignIn/index.js

```diff
@@ -5,6 +5,8 @@ import SignInForm from "./SignInForm";
 /** @jsx jsx */
 import { css, jsx } from "@emotion/core";
 import { NavLink } from "react-router-dom";
+import {login } from "../../../state/ducks/auth";
```

Comment:
```suggestion
import { login } from "../../../state/ducks/auth";
```
We must not have prettier in this project ;)